pkg/selinux/selinux.go

```diff
@@ -270,19 +269,31 @@ func SelinuxGetEnforceMode() int {
 }
 
 func mcsAdd(mcs string) error {
-	if mcsList[mcs] {
-		return fmt.Errorf("MCS Label already exists")
+	filename := fmt.Sprintf("%s/%s", "/run/rkt/", mcs)
+	file, err := os.OpenFile(filename, os.O_CREATE|os.O_EXCL|os.O_RDONLY, 0644)
+	if err != nil {
+		if os.IsExist(err) {
+			return fmt.Errorf("MCS Label already exists")
+		} else {
+			return fmt.Errorf("Unable to test MCS: %s", err)
```

Comment:
Errors should be returned lower-cased. Also, no callers of this function are actually checking them.